### ISSUE TITLE
Evangelion Jo missing a map

### ISSUE THREAD

**Issue by daniel229** [hrydgard/ppsspp#8534]
Since https://github.com/hrydgard/ppsspp/pull/8461
![1](https://cloud.githubusercontent.com/assets/3481559/12700770/c7326954-c82a-11e5-967a-18a1ba558cad.png)
![2](https://cloud.githubusercontent.com/assets/3481559/12700771/c739181c-c82a-11e5-878c-06cdfab19573.png)

GE debugger
![3](https://cloud.githubusercontent.com/assets/3481559/12700767/b86c7388-c82a-11e5-9c28-dfae9d79a00f.png)

https://gist.github.com/daniel229/1870a415c2e36872bc6e

**Comment by unknownbrackets** [comment]
Min Z	004000
Max Z	007fff

We were ignoring those previously.

Clip enable 1

Not sure if important, since minz/maxz are set anyway.

Depth test pass if src >= dst

The z values are 16383.569336, or in other words 0x3FFF.  Maybe these ought to round up.  Does pixel depth rounding or vertex depth rounding help?

-[Unknown]

**Comment by daniel229** [comment]
pixel depth rounding or vertex depth rounding does not help it。

**Comment by daniel229** [comment]
This also broken in softgpu.

**Comment by unknownbrackets** [comment]
Yeah, softgpu doesn't do the necessary rounding either.  Hmm, interesting.  I need to do more tests to be completely sure of the rounding, but I think I was seeing it round up at 0.5.

-[Unknown]

**Comment by unknownbrackets** [comment]
It seems like values in the middle mostly round up.  But I'm getting somewhat strange results.

A few notes:
 * -0.99 through 65535.99 are drawn with clamping off and minz=0, maxz=65535.
 * 0.51 through 65534.99 are drawn with clamping off and minz=1, maxz=65534.
 * <= 0.5 rounds to 0, 1.5 rounds to 1, 2.5 rounds to 2...
 * But 4660.5 rounds up to 4661, and 4661.5 rounds up to 4662 also.
 * 65534.99 rounds to 65534.

-[Unknown]

**Comment by unknownbrackets** [comment]
A frame dump of this scene / bug would be helpful.

-[Unknown]
